### PR TITLE
Implement sonic_gnoi gRPCs in simulator - 14.0.x

### PR DESCRIPTION
**Pull request by marekzatko**
Add some basic implementation of sonic_gnoi gRPCs
 so we have something to test gNOI implementation
  against.

Signed-off-by: marekzatko <Marek.Zatko@pantheon.tech>

### CODE REVIEW

**Review by PeterSuna — COMMENTED** [review]
I don't like these full defined path to object in code:
https://github.com/PANTHEONtech/lighty/pull/736/files#diff-9f9c665ba2ba38c22f0d91d1834ff5a83e85af3468edcb023517fa86b80be265R25

It would be better, if the path is defined already in import and in code is used only object.
import gnoi.sonic.SonicGnoi.TechsupportRequest;
...
showTechsupport(TechsupportRequest request,  StreamObserver\<TechsupportResponse\> responseObserver) {...

File: lighty-modules/lighty-gnmi/lighty-gnmi-device-simulator/README.md [hunk 1]
```diff
@@ -162,3 +162,30 @@ This example will show how to execute basic operations on lighty.io gNMI device
      ]
    }
    ```
+   
+   ## gNOI
+   Simulator implements the following gNOI gRPCs with implemented logic:
```
Comment (vainyksi) [inline lighty-modules/lighty-gnmi/lighty-gnmi-device-simulator/README.md:167]: Simulator implements the following gNOI gRPCs:

File: lighty-modules/lighty-gnmi/lighty-gnmi-device-simulator/README.md [hunk 2]
```diff
@@ -162,3 +162,30 @@ This example will show how to execute basic operations on lighty.io gNMI device
...
+   - os.proto:
+     - install
+   - system.proto:
+     - reboot
+   - sonic.proto:
+     - showTechsupport
+     - copyConfig
+     - imageInstall
+     - imageRemove - returns error (used for testing purposes)
+     - imageDefault
```
Comment (vainyksi) [inline lighty-modules/lighty-gnmi/lighty-gnmi-device-simulator/README.md:190]: until they are supported & implemented, it is not needed to mention them here

**Review by vainyksi — CHANGES_REQUESTED** [review]

File: lighty-modules/lighty-gnmi/lighty-gnmi-device-simulator/README.md (hunk 1 above)
Comment (marekzatko) [reply lighty-modules/lighty-gnmi/lighty-gnmi-device-simulator/README.md]: Done

File: lighty-modules/lighty-gnmi/lighty-gnmi-device-simulator/README.md (hunk 2 above)
Comment (marekzatko) [reply lighty-modules/lighty-gnmi/lighty-gnmi-device-simulator/README.md]: Done

**Comment by marekzatko** [comment]
@PeterSuna 
agree, fixed

**Review by vainyksi — APPROVED** [review]

**Pull request merged**